### DISCUSSION TITLE
Install in a existing partition

### DISCUSSION THREAD

**Discussion by dennisr40** [rhinstaller/anaconda#4133] [Ideas]
As far as I know, it is only possible to install on empty hard disk space.
I would be a good feature to allow installing in a existing partition without deleting the user files.

**Comment by jkonecny12** [comment]
Hi @dennisr40 thanks for proposal. It is possible to re-use existing partitions in the `Custom partitioning` spoke. We require formatting only for some partitions like `/` (root) partition (which will most probably not work otherwise) .

**Reply by dennisr40** [reply]
The root partition and /home/ partition is the same partition on my computer.

**Reply by jkonecny12** [reply]
In that case it's not something we want to support. Erasing a partition is well defined and simple operation but removing specific mount points content is not. There could be a lot of issues happening and it's also a bit ugly operation which should be avoided.

I don't think that benefit of this approach would be so high compared to issues it could bring. Also it's not a recommended way of partitioning on Fedora / CentOS / RHEL.

**Comment by dennisr40** [comment]
I used the automatic partitioning feature of Fedora 36 Workstation. It does not create two partitions for /home/ and / root

**Reply by jkonecny12** [reply]
It depends on version you used but right now on Fedora it creates BTRFS with subvolumes for each of these. These should be detected by the system installation and can be re-used again in the Custom partitioning.
